apache/docker-entrypoint.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e
 
-if [[ "$1" == apache2* ]]; then
+if [[ "$1" == apache2* ]] || [[ "$1" == php-fpm ]]; then
```

Comment:
The new test can be a simple `[ "$1" == php-fpm ]` since it doesn't have a wildcard in it.